src/Parser/Core/Modules/Items/Prydaz.js

```diff
@@ -22,7 +24,11 @@ class Prydaz extends Module {
   item() {
     return {
       item: ITEMS.PRYDAZ_XAVARICS_MAGNUM_OPUS,
-      result: this.owner.formatItemHealingDone(this.healing),
+      result: (
+        <dfn data-tip={`The total damaged absorbed by Archimonde's Hatred Reborn was ${this.owner.formatItemAbsorbDone(this.healing)}.`} >
```

Comment:
A bit nitpicky maybe, but I think this doesn't really give the user any more information so could be removed for the sake of keeping the amount of tooltips to a minimum so they actually read the ones that matter.